File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnFieldRefSet.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnEjectionPolicy;
import com.neome.java.api.meta.base.Types.MetaIdField;
import com.neome.java.api.meta.base.Types.MetaIdSpreadsheet;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldRefSet extends DefnFieldEditable
{
  @Nullable
  public Boolean allowDuplicateValues;

  @Nullable
  public MetaIdField displayFieldId;

  @Nullable
  public EnumDefnEjectionPolicy ejectionPolicy;

  @Nullable
  public DefnLayoutGrid layoutSpreadsheet;

  @Nullable
  public Long maxSize;

  public MetaIdSpreadsheet spreadsheetId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldRefSet.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnFieldEditable
import com.neome.api.meta.base.dto.DefnLayoutGrid
import com.neome.api.meta.base.Types.EnumDefnEjectionPolicy
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdSpreadsheet

interface DefnFieldRefSet : DefnFieldEditable
{
  val allowDuplicateValues: Boolean?
  val displayFieldId: MetaIdField?
  val ejectionPolicy: EnumDefnEjectionPolicy?
  val layoutSpreadsheet: DefnLayoutGrid?
  val maxSize: Long?
  val spreadsheetId: MetaIdSpreadsheet
}